meson.build

```diff
@@ -0,0 +1,162 @@
+                 'generated/gtkd/glib/Str.d',
+                 'generated/gtkd/glib/StringChunk.d',
+                 'generated/gtkd/glib/StringG.d',
+                 'generated/gtkd/glib/TestLogBuffer.d',
+                 'generated/gtkd/glib/TestLogMsg.d',
+                 'generated/gtkd/glib/TestSuite.d',
+                 'generated/gtkd/glib/Thread.d',
+                 'generated/gtkd/glib/ThreadPool.d',
+                 'generated/gtkd/glib/Timeout.d',
+                 'generated/gtkd/glib/Timer.d',
+                 'generated/gtkd/glib/TimeVal.d',
+                 'generated/gtkd/glib/TimeZone.d',
+                 'generated/gtkd/glib/TrashStack.d',
+                 'generated/gtkd/glib/Unicode.d',
+                 'generated/gtkd/glib/UnixUtils.d',
+                 'generated/gtkd/glib/URI.d',
+                 'generated/gtkd/glib/Util.d',
+                 'generated/gtkd/glib/Uuid.d',
+                 'generated/gtkd/glib/VariantBuilder.d',
+                 'generated/gtkd/glib/Variant.d',
+                 'generated/gtkd/glib/VariantDict.d',
+                 'generated/gtkd/glib/VariantIter.d',
+                 'generated/gtkd/glib/VariantType.d',
+                 'generated/gtkd/glib/Version.d',
+                 'generated/gtkd/glib/c/functions.d',
+                 'generated/gtkd/glib/c/types.d',
+                 #GObject
+                 'generated/gtkd/gobject/Binding.d',
+                 'generated/gtkd/gobject/Boxed.d',
+                 'generated/gtkd/gobject/CClosure.d',
+                 'generated/gtkd/gobject/Closure.d',
+                 'generated/gtkd/gobject/Enums.d',
+                 'generated/gtkd/gobject/Flags.d',
+                 'generated/gtkd/gobject/ObjectClass.d',
+                 'generated/gtkd/gobject/ObjectG.d',
+                 'generated/gtkd/gobject/ParamSpec.d',
+                 'generated/gtkd/gobject/ParamSpecPool.d',
+                 'generated/gtkd/gobject/Signals.d',
+                 'generated/gtkd/gobject/TypeClass.d',
+                 'generated/gtkd/gobject/Type.d',
+                 'generated/gtkd/gobject/TypeInstance.d',
+                 'generated/gtkd/gobject/TypeInterface.d',
+                 'generated/gtkd/gobject/TypeModule.d',
+                 'generated/gtkd/gobject/TypePluginIF.d',
+                 'generated/gtkd/gobject/TypePluginT.d',
+                 'generated/gtkd/gobject/TypeValueTable.d',
+                 'generated/gtkd/gobject/ValueArray.d',
+                 'generated/gtkd/gobject/Value.d',
+                 'generated/gtkd/gobject/WeakRef.d',
+                 'generated/gtkd/gobject/c/functions.d',
+                 'generated/gtkd/gobject/c/types.d',
+                 # Mixing generated code and non generated code...argh
+                 'src/gtkd/Implement.d',
+                 'src/gtkd/Loader.d',
+                 'src/gtkd/paths.d',
+                 # This should be split it in different folders
```

Comment:
Ditto. (generated/gtkd/gtkc)

This one is for backwards compatibility, the actual file is already in its own folder. (generated/glib/c/functions.d)